webextensions/api/webRequest.json

```
@@ -1445,11 +1445,9 @@
                 "version_added": "14"
               },
               "firefox": {
-                "version_added": "45"
-              },
-              "firefox_android": {
-                "version_added": "48"
+                "version_added": "138"
               },
+              "firefox_android": "mirror",
```

We should note that the method was exposed but did not function as expected. I have not tested this suggested JSON, but it should follow the format described in 
the [compat data schema](https://github.com/mdn/browser-compat-data/blob/c6992676d95f20b9d9be3a06726cd609e8a2800d/schemas/compat-data-schema.md) docs.

```suggestion
              "firefox": [
                {
                  "version_added": "138"
                },
                {
                  "version_added": "45",
                  "partial_implementation": true,
                  "note": "Exposed but non-functional"
                }
              ]
              "firefox_android": [
                {
                  "version_added": "138"
                },
                {
                  "version_added": "48",
                  "partial_implementation": true,
                  "note": "Exposed but non-functional"
                }
              ],
```

---

@dotproto almost, it's "notes" not "note"